package.json

```diff
@@ -14,6 +14,7 @@
     },
     "devDependencies": {
         "@types/jest": "^24.0.11",
+        "@types/node": "^11.13.0",
```

Comment:
This is too recent; this should match the node version package.json/azure-pipelines.yml use

Reply:
Its actually Type definitions for Node.js 8.10.
<img width="1081" alt="Screen Shot 2019-04-04 at 11 24 51 AM" src="https://user-images.githubusercontent.com/4496335/55579141-73f0f880-56cc-11e9-8ae6-147d2dfbc594.png">